index.js

```diff
@@ -37,13 +38,15 @@ function checkGuess() {
 
     submitButton.disabled = true;
     guessInput.disabled = true;
+
+    console.log(messages[elementIndex]) // added console log 
```

Comment:
Remove this if you don't need it, since it's causing an error in the console when submitting the correct guess. _PS: To console this, it needed to be done within the hideAllMessages function, as outside of that function, you can't access messages using elementIndex."_